src/app/models/Item/index.ts

```diff
@@ -0,0 +1 @@
+export { Item, type IItem } from './Item'
```

Comment:
I don't understand this file. This is for re-exporting the same Items but with different name ?

Reply:
The index file acts as a single entry point for all the exports in that module, making it easier to import everything you need with a single statement.